### PR TITLE
{bio}[GCC/12.2.0] Archive-Zip v1.68, Bio-DB-HTS v3.01, Compress-Raw-Zlib v2.202, ...

### PR DESCRIPTION
**Pull request by stevenvdb**
(created using `eb --new-pr`)

Fairly simple update of VEP, also needed an easyblock update (https://github.com/easybuilders/easybuild-easyblocks/pull/3121)

### CODE REVIEW

**Comment by stevenvdb** [comment]
Is there a way to run the tests taking into account the proposed changes in https://github.com/easybuilders/easybuild-easyblocks/pull/3121? Or should I wait until that one gets merged?

**Comment by ocaisa** [comment]
@boegelbot: please test @ generoso
EB_ARGS=" --include-easyblocks-from-pr=3121"

**Review by ocaisa — APPROVED** [review]

**Comment by ocaisa** [comment]
@boegelbot: please test @ jsc-zen3
EB_ARGS=" --include-easyblocks-from-pr=3121"

**Review by ocaisa — APPROVED** [review]

**Pull request merged**
